server/test/integration/rakTest/handleButtonPressTest.js

```diff
@@ -396,4 +396,70 @@ describe('rak.js integration tests: handleButtonPress', () => {
       expect(buttonAlerts.handleValidRequest).not.to.be.called
     })
   })
+
+  describe('POST Aw== (Button 3 V2) with secondary API key for existing Button', () => {
```

Comment:
I am getting 'Mw==' for 3 encoded into base64

Reply:
Fixed this!